schemas/risk.yml

```diff
@@ -1,40 +1,52 @@
 ---
-- name: hash
+- name: risk
   title: Group
   group: 2
   type: group
-  short: Risk Metrics and Labels
+  short: Risk Scores and Labels
   description: >
-    The risk fields represent the criticality of assets or the potential risk of threats & vulnerabilities
+    Risk fields represent the normalized comparative sensitivity of internal resources or the potential risk of threats, vulnerabilities, and IOCs.
 
-    Field names for common hashes (e.g. MD5, SHA1) are predefined. Add fields for other hashes
-    by lowercasing the hash algorithm name and using underscore separators as appropriate
-    (snake case, e.g. sha3_512).
+    At the event level normalized risk is calculated based on any risk metrics available for lookup for the given event
+    (e.g. a user, source, destination in a firewall log, a client and file in a endpoint file/process log). The final event score
+    is calculated by summing the squares of each avaialble risk score and dividing by the total multiple of available scores.
+    As an example:  ( user.risk + source.risk + destination.risk ) / ( user.risk + source.risk + destination.risk )
+
+    Risk labels are for labeling the type of business risk represented by a given resource, typically in reference to the data
+    housed in an asset or zone, the level of access of a given user or group, or the risk represented by a given threat or IOC.
 
   reusable:
     top_level: false
     expected:
+      - event
+      - user
+      - group
+      - client
+      - source
+      - server
+      - destination
+      - cloud
+      - container
+      - network
       - file
-      - process
+      - threat
+      - vulnerbility
+
 
   fields:
 
-    - name: md5
+    - name: label
```

Comment:
@dainperkins Is there another name we could use instead of "label" ?  As you know, one ECS base field is "labels" (plural), but I am worried about creating confusion, so I wonder if we could use something like `.name` to  be consistent with other field sets?

Reply:
updated to risk.tag for clarity